frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -211,6 +212,31 @@ const baseImageUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions)
   );
 };
 
+const windowsToolsUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
```

Comment:
looks good, but we should move this updater to `storage-tab-state-update.ts` so the relevant will be together

Reply:
moved to storage :+1: